pyproject.toml

```
@@ -33,6 +33,9 @@ dev = ["mypy==1.6.1", "pre-commit==3.5.0", "ruff==0.1.5", "pytest==7.4.3"]
 [project.urls]
 Repository = "https://github.com/rogiervandergeer/pydargs"
 
+[tool.mypy]
+python_version = "3.10"
```

```suggestion
python_version = "3.9"
```

You mentioned earlier this needs to point to the lowest version we support and that is 3.9 in your matrix above

---

Since I added a test for the `UnionType` we need mypy to parse like it is version 3.10 - or otherwise it doesn't like the syntax in the test. The alternative would be to add `type: ignore` to those lines, which isn't ideal either.